engine/src/test/java/com/google/android/fhir/search/SearchTest.kt

```diff
@@ -206,6 +211,232 @@ class SearchTest {
       )
   }
 
+  @Test
+  fun search_token_filter_by_coding_type() {
```

Comment:
```suggestion
  fun search_filter_token_coding() {
```

same for all the test cases below